logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -103,12 +104,15 @@ private enum class CommitStrategy {
     ABORT
 }
 
-private fun CoreFailure.getStrategy(retryOnClientMismatch: Boolean = true): CommitStrategy {
+private fun CoreFailure.getStrategy(
+    retryOnClientMismatch: Boolean = true,
+    retryOnStaleMessage: Boolean = true
```

Comment:
Nice that we have these as configurable, I think is much better to have them like in kalium config? What do you think?

Reply:
I don't think we need them configurable in the kalium config. The reason I made them configurable now is because we need. different behaviour in different scenarios:

- When establishing an MLS group it doesn't make sense to retry on StaleMessage, because that means someone else was faster an established the group before you.
- When adding a participant it makes sense to retry on StaleMessage because it means someone slipped in a another commit before you but that person didn't necessarily do the same change you attempted.